tests/phpunit/unit/api/actions/beansSetAction.php

```diff
@@ -0,0 +1,143 @@
+<?php
```

Comment:
Ignore this function.  It's part of PR #92.  It's shown here as I built this PR off of PR #92's branch.